src/fr/devsylone/fallenkingdom/display/ScoreboardDisplayService.java

```diff
@@ -80,6 +98,23 @@ public boolean containsAny(@NotNull PlaceHolder... placeHolders) {
         return false;
     }
 
+    /**
+     * Vérifie si le titre contient des placeholders PAPI
+     * @return true si le titre contient des placeholders PAPI
+     */
+    public boolean titleContainsPapiPlaceholders() {
+        return this.titleHasPapiPlaceholders;
+    }
+
+    /**
+     * Vérifie si une ligne contient des placeholders PAPI
+     * @param line l'index de la ligne
+     * @return true si la ligne contient des placeholders PAPI
+     */
+    public boolean lineContainsPapiPlaceholders(int line) {
+        return line >= 0 && line < this.linesPapiPlaceholders.size() && this.linesPapiPlaceholders.get(line);
```

Comment:
```suggestion
    private boolean lineContainsPapiPlaceholders(int line) {
        return this.linesPapiPlaceholders.get(line);
```
(Cette méthode n'est jamais appelée) Vu que cette méthode est a priori interne, on peut réduire la complexité de la méthode en omettant la vérification de l'intervalle de l'index (c'est aussi plus utile pour déboguer d'avoir une véritable exception quand il est hors du tableau à mon avis).